neo/rawio/spike2rawio.py

```diff
@@ -608,10 +608,10 @@ def get_sample_interval(info, chan_info):
     Get sample interval for one channel
     """
     if info['system_id'] in [1, 2, 3, 4, 5]:  # Before version 5
-        sample_interval = (chan_info['divide'] * info['us_per_time'] *
+        sample_interval = (int(chan_info['divide']) * info['us_per_time'] *
                            info['time_per_adc']) * 1e-6
     else:
-        sample_interval = (chan_info['l_chan_dvd'] *
+        sample_interval = ((chan_info['l_chan_dvd']) *
```

Comment:
Hi @Warfley!
Couldn't the overflow occur also in this line? Is there a reason you are not introducing type casting for the `else` case?

Reply:
You are absolutely right, I've added also a cast there but somehow it got lost during some git changes (probably when I changed the commit message I accidentally added a temporary change of my working copy). Fixed this mistake